lib/site_prism/waiter.rb

```diff
@@ -14,5 +14,13 @@ def self.wait_until_true(wait_time = Capybara.default_max_wait_time)
 
       raise SitePrism::TimeoutError, "Timed out after #{wait_time}s."
     end
+
+    def self.wait_until_displayed(*args, url)
```

Comment:
This method should essentially house 95% of the logic. And doesn't need to leak state from the original method call. Remember both of these are essentially auto-loaded in